config/locales/en-GB.yml

```diff
@@ -2098,7 +2098,7 @@ en-GB:
     site:
       edit_tooltip: Edit the map
       edit_disabled_tooltip: Zoom in to edit the map
-      createnote_tooltip: Add a note to the map
+      createnote_tooltip: Add a note to the map.
```

Comment:
Please don't make changes to `en-GB.yml` or any other locale file other than `en.yml` as translations need to be done on Translatewiki not here.